pom.xml

```diff
@@ -18,6 +18,10 @@
             <id>public.onebusaway.org</id>
             <url>http://nexus.onebusaway.org/content/groups/public/</url>
         </repository>
+        <repository>
```

Comment:
Here's the repo I was referring to:

~~~
<id>scm.umiacs.umd.edu</id>
<url>https://scm.umiacs.umd.edu/maven/adapt/content/repositories/releases/</url>
~~~